content/bn/security-chaos-engineering.md

```diff
@@ -0,0 +1,22 @@
+---
+title: নিরাপত্তা বিশৃঙ্খলা ইঞ্জিনিয়ারিং (Security Chaos Engineering)
+status: Completed
+category: ধারণা
+tags: ["security", "methodology", ""]
+---
+
+## এটা কি
+
+নিরাপত্তা বিশৃঙ্খলা ইঞ্জিনিয়ারিং অথবা SCE [বিশৃঙ্খলা ইঞ্জিনিয়ারিং](/bn/chaos-engineering/) এর উপর ভিত্তি করে একটি নিয়মানুবর্তিতা। কোলাহলপূর্ণ এবং দূষিত পরিস্থিতি সহ্য করার জন্য সিস্টেমের ক্ষমতার উপর আস্থা তৈরি করতে SCE একটি ডিসট্রিবিউটেদ সিস্টেমে সক্রিয় নিরাপত্তা পরীক্ষা করে (proactive security experimentation)। নিরাপত্তা বিশৃঙ্খলা ইঞ্জিনিয়াররা এটি অর্জন করতে বৈজ্ঞানিক পদ্ধতি লুপ ব্যবহার করে, যার মধ্যে রয়েছে স্থির-স্থিতি (steady-state), হাইপোথিসিস, ক্রমাগত যাচাইকরণ, শিক্ষামূলক অভিজ্ঞতা (lesson learned) এবং প্রশমন (mitigation) বাস্তবায়ন।
+
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
+
+[সাইট নির্ভরযোগ্যতা প্রকৌশলী](/bn/site-reliability-engineering/) (SREs) এবং সাইবার নিরাপত্তা প্রকৌশলীদের প্রধান অগ্রাধিকার হল শূন্য ডাউনটাইম অর্জন এবং ব্যবসায়িক প্রভাব কমানোর লক্ষ্যে যত দ্রুত সম্ভব পরিষেবা পুনরুদ্ধার করা। SRE এবং সাইবার সিকিউরিটি ইঞ্জিনিয়াররা প্রাক-ব্যর্থতা (pre-failure) এবং পরবর্তী ব্যর্থতার ঘটনা উভয় পরিস্থিতির সাথেই মোকাবেলা করে। বেশিরভাগ সুরক্ষা সমস্যাগুলি দ্রুত আবিষ্কার এবং প্যাচ করা চ্যালেঞ্জিং, যেগুলো অ্যাপ্লিকেশন বা সিস্টেম কার্যকারিতাকে প্রভাবিত করে। উপরন্তু, নিরাপত্তা ঘটনাগুলি সাধারণত ডেভেলপমেন্টের সময় চিহ্নিত করা কঠিন।
```

Comment:
Kindly make a moderate sentence  instead of this "বেশিরভাগ সুরক্ষা সমস্যাগুলি দ্রুত আবিষ্কার এবং প্যাচ করা চ্যালেঞ্জিং, যেগুলো অ্যাপ্লিকেশন বা সিস্টেম কার্যকারিতাকে প্রভাবিত করে"

Reply:
Done